t/ids_from_pages.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+describe 'Wikidata#ids_from_pages' do
+  around { |test| VCR.use_cassette(titles.first, &test) }
+  subject { WikiData.ids_from_pages('en', titles) }
+
+  describe 'Direct links' do
+    describe 'single title' do
+      let(:titles) { ['Douglas Adams'] }
+      it 'pairs a title with an id' do
+        subject.must_equal('Douglas Adams' => 'Q42')
+      end
+    end
+
+    describe 'multiple titles' do
+      let(:titles) { ['Towel', 'Douglas Adams'] }
+      it 'pairs an id with each title' do
+        subject.must_equal('Douglas Adams' => 'Q42', 'Towel' => 'Q131696')
```

Comment:
… especially as the results are given here in the reverse order to the `titles` array.

Each of these go against the Principle of Least Surprise, and combination makes it appear that there's something not-entirely-obvious being tested here, that the reader needs to pay much closer attention to than is really warranted.